content/contributor-guidelines.md

```diff
@@ -0,0 +1,52 @@
+
+We have recently re-started a [wiki], and eagerly encourage gardening. While the Zulip does have archives, they are not indexed by search engines (though search is available within Zulip), so the wiki represents the definitive place for the results of discussions and resources.
+
+To propose a nontrivial change, it is better to file an issue first rather than sending a PR. Depending on the scope and disruption of the change, it might make sense to bring it up on Zulip. Especially if a clear consensus doesn’t emerge in a reasonable time, discuss it in office hours.
+
+While the main focus of the Linebender community infrastructure is to develop Xilem, Vello, and related modules, we also want to help move the entire ecosystem forward. People from other Rust UI efforts are welcome, and discussions about how to join forces and share infrastructure especially so.
+
+We use the Rust Code of Conduct and hold people to a much higher standard of conduct than is usual on the Internet. Don’t worry if you’re interacting in good faith, though, the moderators will work with you to try to resolve issues.
+
+## Code reviews
+
+Here are our policies around code review:
+
+Every PR must be approved by someone with write access before committing.
+The [wiki] is an exception; there we optimistically merge without review
+Usually we ask the original author of the PR to commit if they have write access.
+Both merge and squash commits are fine, it’s up to the committer.
+We give write access to everyone with a good faith intent to improve the code. Usually this is shortly after the first commit.
+There are currently five owners: @raphlinus, @dfrg, @DJMcNab, @xStrom, and @cmyr. They approve new committers and can make changes to the organization.
+
+We struggle with getting PRs approved and landed quickly. To try to improve that, here are some suggestions.
+
+First, we encourage everybody to review PRs. It’s a good way for newcomers to get up to speed on the project, and contribute in a meaningful way. There are two goals of the code review process: to improve the quality of the code that lands, and for people to learn something; it’s worthwhile if some knowledge flows in either direction or ideally both.
+
+If a PR goes unreviewed too long, it’s absolutely a good idea to ping it on Zulip. It’s also fine to ask for reviewers during office hours, especially if there’s a deeper issue to be discussed.
+
+## Responsibility
+
+Linebender is a collection of open source projects, with a research flavor. We make no guarantees of any of them being suitable for production. We welcome contributions from anybody. Participating should be done out of joy and should not feel like a burden.
+
+Sometimes PRs stall out, or people have an intent to work on an issue and don’t get to it. In those cases, we encourage others to take them over. We try to discourage “cookie licking” as much as possible, though it does happen (especially with Raph).
```

Comment:
```suggestion
Sometimes PRs stall out, or people have an intent to work on an issue and don’t get to it. In those cases, we encourage others to take them over. We try to discourage “cookie licking” as much as possible.
```

I think we can omit the joke here. Especially as someone new reading this won't know that Raph himself wrote this. We don't really have a culture of taking jabs at fellow contributors, but this may look like it.